feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/balance/common/AssetExpandableAssetDecorationSettings.kt

```diff
@@ -0,0 +1,6 @@
+package io.novafoundation.nova.feature_assets.presentation.balance.common
+
+import android.view.animation.AccelerateDecelerateInterpolator
+import io.novafoundation.nova.common.utils.recyclerView.expandable.ExpandableAnimationSettings
+
+fun createAssetExpandableAnimationSettings() = ExpandableAnimationSettings(400, AccelerateDecelerateInterpolator())
```

Comment:
I suggest to put such "factory" functions as extensions to Companion object of the target create:

``` kotlin
ExpandableAnimationSettings.createForAssets()
```

This way we would not "pollute" the global namespace so much